web/gds-admin-ui/src/pages/app/details/BasicDetails/components/BasicDetailsDropdown.js

```diff
@@ -42,9 +43,18 @@ const BasicDetailsDropDown = ({ isNotPendingReview, vasp }) => {
                 <i className="dripicons-dots-3"></i>
             </Dropdown.Toggle>
             <Dropdown.Menu>
-                <Dropdown.Item data-testid="reviewItem" disabled={isNotPendingReview()}>
-                    <i className="mdi mdi-card-search me-1"></i>Review
-                </Dropdown.Item>
+                <Modal>
+                    <ModalOpenButton>
+                        <Dropdown.Item data-testid="reviewItem" disabled={isNotPendingReview()}>
```

Comment:
Right now we have two methods of detecting if a review is possible; `isNotPendingReview` and a 404 from `GET /v2/vasps:/id/review`; these seem to be conflicting with each other. I'll create a story to specify how these should interact.